index.js

```diff
@@ -161,6 +166,47 @@ const getRetrieval = async (req, res, client, retrievalId) => {
   })
 }
 
+const getRoundDetails = async (req, res, client, getCurrentRound, roundParam) => {
+  const roundNumber = await parseRoundNumberOrCurrent(getCurrentRound, roundParam)
+
+  if (roundParam === 'current') {
+    res.setHeader('cache-control', 'no-store')
+  }
+
+  const { rows: [round] } = await client.query('SELECT * FROM spark_rounds WHERE id = $1', [roundNumber])
+  if (!round) {
+    return notFound(res)
+  }
+
+  const { rows: tasks } = await client.query('SELECT * FROM tasks WHERE round_id = $1', [roundNumber])
+
+  json(res, {
+    roundId: roundNumber.toString(),
+    endsBefore: new Date(round.created_at.getTime() + ROUND_LENGTH_IN_MS),
+    tasks: tasks.map(it => ({
```

Comment:
what does `it` stand for here?

Reply:
It stands for whatever we are iterating over - the element we are mapping.

I was using `t`, but then I realised it's potentially confusing because we use `t` for the current test instance in TAP-like tests too.

I also don't want to use `task` because it's too similar to `tasks`.

What name do you use in cases like this?

Reply:
Interesting! To me `it` means iterable/iterator. See for example https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols?retiredLocale=de#simple_iterator

In this case no one is right of course. I think I would have used `task` or `t`. I don't feel strongly about this

Reply:
Renamed all `it` variables to `t` in 8b925b3